atlas/templates/helm/templates/pubsub-deploy.yaml.gotmpl

```diff
@@ -0,0 +1,47 @@
+{{ "{{" }} if .Values.pubsub.enabled {{ "}}" }}
```

Comment:
I don't like an idea to copy deployments for pubsub into charts, we should consider an option to set this as a dependency